pylops/basicoperators/CausalIntegration.py

```diff
@@ -7,14 +9,21 @@
 class CausalIntegration(LinearOperator):
     r"""Causal integration.
 
-    Apply causal integration to a multi-dimensional array along ``dir`` axis.
+    Apply causal integration to a multi-dimensional array along ``axis``.
 
     Parameters
     ----------
     dims : :obj:`list` or :obj:`int`
         Number of samples for each dimension
+    axis : :obj:`int`, optional
+        .. versionadded:: 2.0.0
+
+        Axis along which the model is integrated.
     dir : :obj:`int`, optional
-        Direction along which array is integrated.
+
```

Comment:
General comment for all operators. As we are doing with PR targeting v2.0.0 (this will never make it into any v1 versions, shall we really keep `dir` as an input parameter and allow people to still use it?

Reply:
That is a good point. I say we just get rid of `dir`, `dirs`  and `nodir` entirely then. I will start working on it this week.